### PR TITLE
Use Esri vector basemaps

### PR DESCRIPTION
**Pull request by michaelpnelson**
link to demo deployment of your smk based app is 
 https://smk-mrdt-rt-b16795-dev.apps.silver.devops.gov.bc.ca

Verify that it looks good, if so merge the pull request which will trigger a prod deployment, otherwise close the pull request without merging

 mentions / notifications: @dgboss, @NicoledeGreef

### CODE REVIEW

**Comment by dgboss** [comment]
The Vanity URL for the dev deploy is https://smk-mrdt-api-gov-bc-ca.test.api.gov.bc.ca
 

If this is a dev deploy,  verify that it looks good, then merge the pull request to trigger prod deployment.

 If this is a prod deployment, view your prod deploy at the url above :)

 mentions / notifications: @dgboss, @NicoledeGreef

File: smk-config.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
             ],
             "zoom": 6.75
         },
-        "baseMap": "Streets"
+        "baseMap": "Streets",
+        "esriApiKey": "AAPK869398b518894ae8aabcb897633e0db3LX46wT9TRnePMwjImmKZcToS2KGrpeJN66cCNnMKdjhkCLefNJn9OYKagquWaF6b"
```
Comment (dgboss) [inline smk-config.json:19]: Are you using a different api key for each map? If not, we should be so we are less likely to bump into the tile limit.

**Review by dgboss — APPROVED** [review]
Changes are fine, I just have the one question about the API key.

File: smk-config.json (hunk 1 above)
Comment (michaelpnelson) [reply smk-config.json:19]: Yes, I'm creating an API key in the keymaker account for each hosted SMK app. I'm also coordinating with Caitlin and waiting on her go-ahead before moving any basemap changes to production.

**Pull request merged**